web/router.go

```diff
@@ -33,7 +33,6 @@ var templatePaths = []string{
 	"template/partial/*.gohtml",
 	"template/partial/stream/*.gohtml",
 	"template/partial/course/manage/*.gohtml",
-	"template/partial/stream/chat/*.gohtml",
```

Comment:
I guess this a follow up to a refactoring PR?

Reply:
Yes. 👍🏻